terraform/modules/elasticache/security_groups.tf

```diff
@@ -0,0 +1,25 @@
+resource "aws_security_group" "redis" {
+  name        = "${var.prefix}-redis-security-group"
+  description = "Redis security group"
+  vpc_id      = var.vpc_id
+
+  ingress {
+    description     = "Allow ingress on port 6379 from anything within the private subnet cidr block range"
+    protocol        = "tcp"
+    from_port       = 6379
+    to_port         = 6379
+    security_groups = var.ingress_source_security_group_ids
+  }
+
+  egress {
+    description = "Allow egress from port 6379 to anything within the private subnet cidr block range"
+    protocol    = "tcp"
+    from_port   = 6379
+    to_port     = 6379
+    cidr_blocks = [var.private_subnet_cidr]

```

Comment:
I think this could be by security group as well, for consistency (although it's less odd to be a range on the way out, we're not expecting this to need egress to the whole vpc).